text-ui-test/savedata.txt

```diff
@@ -0,0 +1,4 @@
+LOG:
+T | [T][✗] here
+E | [E][✗] who (at: ep)
+D | [D][✗] here (by: now)
```

Comment:
Ideally, the save file should be ignored by .gitignore as you don't want git to track such trivial changes